ambari-metrics/ambari-metrics-assembly/pom.xml

```diff
@@ -1296,6 +1296,16 @@
       <groupId>org.apache.ambari</groupId>
       <artifactId>ambari-metrics-timelineservice</artifactId>
       <version>${project.version}</version>
+        <exclusions>
+            <exclusion>
+                <artifactId>jackson-core</artifactId>
+                <groupId>com.fasterxml.jackson.core</groupId>
+            </exclusion>
+            <exclusion>
+                <artifactId>jackson-databind</artifactId>
+                <groupId>com.fasterxml.jackson.core</groupId>
+            </exclusion>
+        </exclusions>
     </dependency>
```

Comment:
This exclusion should not be needed normally. If _ambari-metrics-timelineservice_ does not depend on old versions, yet depending on _ambari-metrics-timelineservice_ brings in old versions, this means something's wrong with the config.

I encountered such error when the _dependencyConfig_ section of the upper level project (probably _ambari-metrics_ in this case) was not set up properly.

Reply:
It looks that we just not need this section at all. It shows transitive dependency as i haven't modified depended modules at time of building dependency graph. Fixed!